src/sentry/options/defaults.py

```diff
@@ -2555,6 +2555,18 @@
     flags=FLAG_PRIORITIZE_DISK | FLAG_AUTOMATOR_MODIFIABLE,
 )
 
+register(
+    "discover.saved-query-dataset-split.enable",
+    default=False,
+    flags=FLAG_PRIORITIZE_DISK | FLAG_AUTOMATOR_MODIFIABLE,
+)
+register(
+    "discover.saved-query-dataset-split.organization-allowlist",
```

Comment:
nit: can we call this
```suggestion
    "discover.saved-query-dataset-split.organization-id-allowlist",
```